### DISCUSSION TITLE
next release date

### DISCUSSION THREAD

**Discussion by anis-campos** [koxudaxi/datamodel-code-generator#2119] [General]
hi @koxudaxi, thanks for merging https://github.com/koxudaxi/datamodel-code-generator/pull/2102.

I'm looking forward for the next release, do you know when will it be ? my team is waiting for it to do some work with the generator.

Thanks in advance.

**Comment by koxudaxi** [comment]
@anis-campos 

There's one point of concern. I was thinking of releasing once this is resolved.
I don't want to make any breaking changes, but could I get your opinion on this? Copy
https://github.com/koxudaxi/datamodel-code-generator/pull/2100#issuecomment-2417303430

**Reply by anis-campos** [reply]
Ho I see. well, I changed the logic of the default value in my PR. I wonder if the assertion made iin the comment is still true, `datetime` is not the default value of the field anymore.

let me check this 🤔

**Reply by anis-campos** [reply]
Ok so now that the both the config `output_datetime_class` and the later argument `target_datetime_class` are optional with default to `None`, I don't think the default changed.

And from what I undersand, the whole point of #2100 can be resumed to https://github.com/koxudaxi/datamodel-code-generator/blob/4d67a81030f9058e9e292f46ca59bbd396d07ac2/datamodel_code_generator/model/pydantic_v2/types.py#L39-L42, so no harm can be done unless you choose to use native or aware.

So I guess I fixed the issue 😄

**Comment by ilovelinux** [comment]
Features of the mentioned PR have been released with [v0.26.2](https://github.com/koxudaxi/datamodel-code-generator/releases/tag/0.26.2), closing :slightly_smiling_face:
